### PR TITLE
Move sustained immunity and viral shedding out of pathogenesis (?)

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

I started editing the antibody section as part of pathogenesis, but I think it will be better to include it in the diagnostics section as part of the background on serological testing, and the viral shedding info didn't seem to quite fit in pathogenesis either.
I did not finish making changes, but wanted to suggest the move in case there are other thoughts about where it should go.
Additional information will need to be added, since I only reviewed one SARS-CoV-2 paper.

### Suggested reviewers (optional)

@cgreene @cbrueffer 

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -154,6 +154,19 @@ On a related note, for some infectious agents an epidemic can be stopped from gr
...
 However, for SARS-CoV-2 and COVID-19, because of the calculated R0 and mortality and the recorded per-region deaths and accumulated cases with the estimated factor of undetected cases, relying on herd immunity without vaccines and/or proven treatment options and/or strong non-pharmaceutical measures of prevention and control would likely result in a significant loss of life.
 
+#### Sustained Immunity to COVID-19
+
+In the process of mounting a response to a viral pathogen, the immune system produces antibodies specific to the pathogen.
+Understanding the acquisition and retention of antibodies is important both to the diagnosis of prior infections that are no longer active and to the development of vaccines.
+The two immunoglobulin classes most pertinent to these areas of interest are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most common antibodies found in the blood.
```
Comment (gregszetoAI) [inline content/10.diagnostics.md:161]:
```suggestion
The two immunoglobulin classes most pertinent to these areas of interest are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies found in the blood.
```

File: content/10.diagnostics.md [hunk 2]
```diff
@@ -154,6 +154,19 @@ On a related note, for some infectious agents an epidemic can be stopped from gr
...
+Following SARS infection, IgM antibodies were first detected in most patients during the second week following infection and remained detectable until the twelfth week, while IgG antibodies were first detected in the second or third week and remained at close to their peak concentration through the thirteen weeks of the study [@doi:10.1056/NEJM200307313490520].
```
Comment (gregszetoAI) [inline content/10.diagnostics.md:162]: Original reference here was a letter to editor. Potential replacement reference:

```suggestion
Following SARS infection, IgM and IgG antibodies were detected in the second week post-infection. IgM titers peaked by the first month post-infection, and then declined to undetectable levels after day 180. IgG titers peaked by day 60, and persisted in all donors through the two-year duration of study. [@doi:10.1111/j.1440-1843.2006.00783.x].
```

File: content/10.diagnostics.md [hunk 3]
```diff
@@ -154,6 +154,19 @@ On a related note, for some infectious agents an epidemic can be stopped from gr
...
+A two-year longitudinal study following convalesced SARS patients with a mean age of 29 found that IgG antibodies were detectable in all 56 patients surveyed for at least 16 months, and remained detectable in all but 4 (11.8%) of patients through the full two-year study period [@doi:10.1086/500469].
+The longevity of antibodies to SARS-CoV-2 remains an ongoing question.
```
Comment (gregszetoAI) [inline content/10.diagnostics.md:164]:
```suggestion
The persistence of antibodies to SARS-CoV-2 remains under investigation. Circulating antibody titers to other coronaviruses have been reported to decline significantly after 1 year. [@doi:10.1017/s0950268800048019]
```

**Review by cgreene — APPROVED** [review]
With @gregszetoAI's changes I approve. I'll file a separate draft PR trying to add @gregszetoAI as a contributor so that we don't forget!

File: content/10.diagnostics.md [hunk 4]
```diff
@@ -154,6 +154,22 @@ On a related note, for some infectious agents an epidemic can be stopped from gr
...
+The persistence of antibodies to SARS-CoV-2 remains under investigation.
+Circulating antibody titers to other coronaviruses have been reported to decline significantly after 1 year [@doi:10.1017/s0950268800048019].
+An early study (initially released on _medRxiv_ on February 25, 2020) presented a chemiluminescence immunoassay to a synthetic peptide derived from the amino acid sequence of the SARS_CoV-2 _S_ protein [@doi:10.1093/infdis/jiaa243].
+This method was highly specific to SARS-CoV-2 and detected IgM in 57.2% and IgG in 71.4% and 57.2% of sera samples from 276 confirmed COVID-19 patients.
+They reported that they could detect IgG within two days of the onset of fever and were not able to detect IgM any earlier, a pattern they compared to findings in MERS.   
+<!--TODO: add https://www.nature.com/articles/s41591-020-0965-6#Sec9 and evalute reports of recurrence. Pausing for now because moving this section out of pathogenesis-->
+
```
Comment (rando2) [inline content/10.diagnostics.md:172]: Incorporating @gregszetoAI's [comment](https://github.com/greenelab/covid19-review/pull/631#discussion_r483424115)
```suggestion
However, although antibody titers often wane over time, these reduced levels do not necessarily indicate a loss of effective immune protection.
For example, memory cells can rapidly recall and produce antibodies on demand in the event of a second infection. 
One study reported an asymptomatic second infection of COVID-19 [@doi:10.1093/cid/ciaa1275].
No antibodies were detected until day 5 after the second infection, suggesting potential immune protection from symptoms. 

```

**Comment by rando2** [comment]
@cgreene if the suggestion I added is a faithful incorporation of @gregszetoAI's comment, then hopefully these suggestions have all been addressed! Thank you both!

**Review by cgreene — APPROVED** [review]
Looks good to me!

**Pull request merged**